### PR TITLE
PER 845 and 1071

### PR DESCRIPTION
**Pull request by dougortiz**
- Removed oidentd service
- Cleaned up help comments for 'setup_pgd' defaults main.yml
- Added 'max_prepared_transactions' variable for 'setup_pgd'

### CODE REVIEW

File: roles/setup_pgd/defaults/main.yml [hunk 1]
```diff
@@ -113,3 +113,27 @@ supported_pg_version:
...
+#  - parent_group: 'pgd_cluster'
+#  - cs_origin_node_group: 'pgd_gc_nodes'
+#  - member_nodes: ['edb-primary3']
+#  - default_group_cs: true
+#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
+
+# For CAMO Commit Scopes
+max_prepared_transactions: 100
+
+pgd_commit_scopes: ""
```
Comment (hannahms) [inline roles/setup_pgd/defaults/main.yml:139]: All of this should be moved in front of the `supported_pgd_version` var.

File: roles/setup_pgd/tasks/pg_hba_config.yml [hunk 2]
```diff
@@ -57,6 +56,52 @@
...
+          "databases": pg_database
+          }
+        ] }}
+  when: pg_ssl
+  loop: "{{ pgd_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
```
Comment (hannahms) [inline roles/setup_pgd/tasks/pg_hba_config.yml:104]: I think we should ask @vibhorkum about these lines, maybe we can discuss how to allow nodes to communicate with one another without using exclusively the replication user.

File: roles/setup_pgd/tasks/setup_pgd.yml [hunk 3]
```diff
@@ -106,3 +106,23 @@
...
+
+- name: Filter items based on cs_type 'GROUP_COMMIT'
+  set_fact:
+    filtered_items_group_commit: "{{ pgd_commit_scopes | selectattr('cs_type', 'equalto', 'GROUP_COMMIT') | list }}"
+
+- name: Display elements of each filtered 'GROUP_COMMIT' item
+  ansible.builtin.include_tasks: pgd_group_commitscope_configuration.yml
+  loop: "{{ filtered_items_group_commit }}"
+  loop_control:
+    label: "{{ item.cs_name }}"
```
Comment (hannahms) [inline roles/setup_pgd/tasks/setup_pgd.yml:128]: These tasks should occur prior to validation tasks in `setup_pgd` role.

File: roles/setup_pgd/README.md
Comment (hannahms) [inline roles/setup_pgd/README.md:1]: Why is it showing all the updates to this file as new?

**Review by hannahms — CHANGES_REQUESTED** [review]
-README rebasing 
-HBA updates (conversation needed)
-movement of items in main.yml and setup_pgd.yml

File: roles/setup_pgd/README.md
Comment (dougortiz) [reply roles/setup_pgd/README.md:1]: Unsure why

File: roles/setup_pgd/defaults/main.yml (hunk 1 above)
Comment (dougortiz) [reply roles/setup_pgd/defaults/main.yml:139]: Will move

File: roles/setup_pgd/tasks/pg_hba_config.yml (hunk 2 above)
Comment (dougortiz) [reply roles/setup_pgd/tasks/pg_hba_config.yml:104]: JIRA ticket created: https://enterprisedb.atlassian.net/browse/PER-1091

File: roles/setup_pgd/defaults/main.yml [hunk 4]
```diff
@@ -111,5 +111,28 @@ supported_pg_version:
...
+#  - member_nodes: ['edb-primary3']
+#  - default_group_cs: true
+#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
 supported_pgd_version:
   - 5
+
+# For CAMO Commit Scopes
+max_prepared_transactions: 100
+
+pgd_commit_scopes: ""
```
Comment (hannahms) [inline roles/setup_pgd/defaults/main.yml:138]: these should also be with the comments and I think they should be moved up some more
```
# The following parameters must be configured in the 'pgd_commit_scopes' list
# the values below are examples and should be adjusted to your requirements
#  for CAMO
# pgd_commit_scopes:
#  - cs_type: 'CAMO'
#  - cs_name: 'camo_scope_1'
#  - parent_group: 'pgd_cluster'
#  - cs_origin_node_group: 'pgd_two_nodes'
#  - member_nodes: ['edb-primary1', 'edb-primary2']
#  - default_group_cs: true
#  - cs_rule: "ALL ( pgd_two_nodes ) ON visible CAMO DEGRADE ON (timeout=500s) TO ASYNC"
#  for GROUP_COMMIT
# pgd_commit_scopes:
#  - cs_type: 'GROUP_COMMIT'
#  - cs_name: 'group_commit_scope_1'
#  - parent_group: 'pgd_cluster'
#  - cs_origin_node_group: 'pgd_gc_nodes'
#  - member_nodes: ['edb-primary3']
#  - default_group_cs: true
#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
pgd_commit_scopes: ""

# when CAMO commit scope present, max_prepared_transactions cannot be 0 (default)
max_prepared_transactions: 100

etc_hosts_lists: []
supported_os:
  - CentOS7
  - CentOS8
  - RedHat7
  - RedHat8
  - Rocky8
  - AlmaLinux8
  - OracleLinux7
supported_pg_type:
  - EPAS
  - PG
supported_pg_version:
  - 14
  - 15
  
 supported_pgd_version:
  - 5
```

**Comment by dougortiz** [comment]
Closed as of 08/24/2023

**Pull request closed without merging**